### PR TITLE
update istio.io@customers doc

### PR DESCRIPTION
**Pull request by jackge007**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by jackge007** [comment]
@googlebot I consent.

File: content/en/about/community/customers/index.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Tons of people are using Istio. Maybe you should too?
     {{< company_logo link="https://www.tuya.com" logo="./tuya.png" alt="Tuya Inc" >}}
     {{< company_logo link="https://www.ibm.com/watson" logo="./watson.png" alt="IBM Watson" >}}
     {{< company_logo link="https://ygrene.com" logo="./ygrene.jpeg" alt="Ygrene Energy Fund" >}}
+    {{< company_logo link="https://iherb.com" logo="./iherb.jpeg" alt="iHerb" >}}
```
Comment (frankbu) [inline content/en/about/community/customers/index.md:37]: Entries are in alphabetical order, this one needs to be moved up. Also, the image is not working, see preview: https://deploy-preview-7597--preliminary-istio.netlify.app/about/community/customers/

Comment (ericvn) [reply content/en/about/community/customers/index.md:37]: Image is failing as the link is specifying .jpeg, but the added file is .svg

**Comment by jackge007** [comment]
update link to svg files

**Comment by frankbu** [comment]
@jackge007 some of the commits are from a different user (gewenjie_iherb) who will also need to sign the CLA.

Also, please move your new entry in the index.md file to the correct location. Entries are in alphabetical order.

**Comment by jackge007** [comment]
would like to fix and commit with another ID with CLA signed.

**Pull request closed without merging**